### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by guidezpl**
Add default issue templates with labels

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (perclasson) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do we also want to encourage users to provide their `flutter doctor -v`?

Comment (rami-a) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should we also ask them to include the flutter doctor output?

Comment (guidezpl) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed :)

Comment (guidezpl) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed :)

**Pull request merged**